tests/phpunit/Admin/WC_REST_Stripe_Settings_Controller_GB_Test.php

```diff
@@ -29,7 +26,7 @@ class WC_REST_Stripe_Settings_Controller_GB_Test extends WC_Mock_Stripe_API_Unit
 	/**
 	 * Gateway instance that the controller uses.
 	 *
-	 * @var WC_Gateway_Stripe
+	 * @var WC_Stripe_UPE_Payment_Gateway
```

Comment:
Non-blocking style note: I prefer using the fully-qualified class name so it's easier to see exactly which class is in use, especially in diffs. I think we should discuss this as a team before making changes to any code, though.
```suggestion
	 * @var \WC_Stripe_UPE_Payment_Gateway
```